devmand/gateway/src/devmand/test/cli/QueuedCliTest.cpp

```diff
@@ -70,6 +72,29 @@ TEST_F(QueuedCliTest, queuedCli) {
   }
 }
 
+TEST_F(QueuedCliTest, queueOrderingTest) {
+  unsigned int iterations = 20;
```

Comment:
I would do more iterations... but not scale threads linearly

Reply:
sure, we can bump it up and use less threads